model_zoo/models/alexnet/model_alexnet.prototext

```diff
@@ -56,15 +56,28 @@ model {
 
   layer {
     name: "data"
-    children: "conv1 target"
+    children: "images labels"
     data_layout: "data_parallel"
     input {
       io_buffer: "partitioned"
     }
   }
+  layer {
```

Comment:
Why are there identity layers added here?  It seems like you can just take the data directly to the consuming layer.